app/src/main/kotlin/com/wire/android/mapper/MessagePreviewContentMapper.kt

```diff
@@ -247,10 +247,20 @@ fun MessagePreview.uiLastMessageContent(): UILastMessageContent {
                 is WithUser.TeamMemberRemoved -> UILastMessageContent.None // TODO
                 is WithUser.Text -> UILastMessageContent.SenderWithMessage(
                     sender = userUIText,
-                    message = UIText.DynamicString((content as WithUser.Text).messageBody),
+                    message = (content as WithUser.Text).messageBody.let { UIText.DynamicString(it) },
                     separator = ": "
                 )
 
+                is WithUser.Composite -> {
+                    val text = (content as WithUser.Composite).messageBody?.let { UIText.DynamicString(it) }
+                        ?: UIText.StringResource(R.string.last_message_composite_with_miossing_text)
+                    UILastMessageContent.SenderWithMessage(
+                        sender = userUIText,
+                        message = text,
+                        separator = ": "
```

Comment:
In the future we could have this separator `": "` as a const from somewhere instead of always hardcoding it, but not for now